ompy/matrix.py

```diff
@@ -218,6 +219,10 @@ def save(self, path: Union[str, Path], filetype: Optional[str] = None,
             filetype = filetype_from_suffix(path)
         filetype = filetype.lower()
 
+        if self.std is not None:
+            warnings.warn(UserWarning(
+                "The `std` attribute of Matrix class cannot be saved to file."))
```

Comment:
Suggestion:   
"The `std` attribute of Matrix class has to be saved to file 'manually'."

Reply:
We could also easily add a `which='values'` or `std` flag for the `save` to make it easier. Does that sound reasonable?

Reply:
Both of your suggestions are great! I will make the changes.

Reply:
Done!